criticalinfrastructure/hsip-api.go

```diff
@@ -32,53 +32,74 @@ const (
-	CellularTowers                      //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Cellular_Towers_New/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	DialysisCenters                     //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Dialysis_Centers/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	EmergencyMedicalServiceStations     //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Emergency_Medical_Service_(EMS)_Stations_gdb/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	EPAandFRSPowerPlants                //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Environmental_Protection_Agency_EPA_Facility_Registry_Service_FRS_Power_Plants/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	ER_TSCA_Facilities                  //404 Error
-	EOA_FRS_WasterwaterTreatmentsPlants //404 Error
-	FacilityInterests                   //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Facility_Interest/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	FerryTerminals                      //https://geo.dot.gov/server/rest/services/Hosted/Ferry_Terminals_DS/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	FireTerminals                       //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Fire_Station/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
-	FMTransmissionTowers                //404 Error
-	GeneratingUnits                     //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/GeneratingUnits1/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
+	// services1.arcgis.com HIFLD additions
+	BRSandEBSTransmitters //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Broadband_Radio_Service_(BRS)_and_Educational_Broadband_Service_(EBS)_Transmitters/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson Broadband Radio Service (BRS) and Educational Broadband Service (EBS) Transmitters
+	CellularTowers        //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Cellular_Towers_New/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
+	DialysisCenters       //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Dialysis_Centers/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
+	EPAandFRSPowerPlants  //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Environmental_Protection_Agency_EPA_Facility_Registry_Service_FRS_Power_Plants/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
+	FacilityInterests     //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/Facility_Interest/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
+	GeneratingUnits       //https://services1.arcgis.com/Hp6G80Pky0om7QvQ/arcgis/rest/services/GeneratingUnits1/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson
 	// Hospitals
```

Comment:
im not sure why hospitals got commented out - it should work.

Reply:
It was already there, same with PowerPlants, but put there to keep the order.